package-coverage/utils/directories.go

```diff
@@ -28,11 +28,14 @@ func FindAllCoverageFiles(basePath string) ([]string, error) {
 func finder(basePath string, searchFor mode) ([]string, error) {
 	found := []string{}
 
+	oldCurrentDirectory := GetCurrentDir()
 	err := os.Chdir(basePath)
 	if err != nil {
 		return nil, err
 	}
 
+	defer os.Chdir(oldCurrentDirectory)
```

Comment:
@corsc this was unintentional, I hope? This needs to go in for the directory enumeration tests which I added to work.